File: java_backup/my java/tapabrata/iisscc/sum1.java
package iisscc;
class sum1
{//beginning of class
public static void main(int a,int n)
{//beginning of main
int i,j,f;//declaring variables
double s=0.0;
for(i=1;i<=n;i++)
{//using for loop
f=1;
for(j=1;j<=i;j++)
f=f*j;
s=s+((double)Math.pow(a,i)/(double)(a+f));//storing sum
}
System.out.println("sum of the series : ");
System.out.println(s);//printing the sum
}//end of main
}//end of class

File: 6th_sem/Systemfiles_Anubhab/FINALMATLAB/sum series/sum1.m
%function s=sum1(n) : Function to calculate s=(1)+(1+2)+...+(1+2+...+n)
function s=sum1(n)
s=0;
s1=0;
  for i=1:n
    s1=s1+i;
    s=s+s1;
  end
end